src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/AttachCaseCallbackService.java

```diff
@@ -130,4 +149,12 @@ private String createEventSummary(
             theCase.getId()
         );
     }
+
+    private boolean isExceptionRecordAttachedToCase(CaseDetails exceptionRecordDetails) {
+        Object attachToCaseReference = exceptionRecordDetails
```

Comment:
what if `exceptionRecordDetails` is null?

Reply:
it is not null - assured by api call in previous step